src/frontend/src/pages/AdminToolsPage/AdminToolsAttendeeDesignReviewModal.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
+import {
+  TextField,
+  FormControl,
+  InputLabel,
+  Select,
+  MenuItem,
+  SelectChangeEvent,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow
+} from '@mui/material';
+import NERModal from '../../components/NERModal';
+
+interface AttendeeDesignReviewModalProps {
+  open: boolean;
+  onClose: () => void;
+}
+
+const AttendeeDesignReviewModal: React.FC<AttendeeDesignReviewModalProps> = ({ open, onClose }) => {
+  const [selectedTeam, setSelectedTeam] = useState('');
+  const [searchQuery, setSearchQuery] = useState('');
+  const tableHeaderCellStyle = {
+    borderRight: '2px solid white',
+    borderBottom: '2px solid white'
+  };
+
+  const tableBodyCellStyle = {
+    borderRight: '2px solid white'
+  };
+
+  const lastHeaderCellStyle = {
+    borderBottom: '2px solid white'
+  };
+
+  // to be deleted later, this is just stub data for filter options
```

Comment:
in the comment can you put "TODO: to be deleted later..."

if we have the word TODO it helps up remember to go back to it to fix it